content/applications/finance/fiscal_localizations/hong_kong.rst

```diff
@@ -0,0 +1,63 @@
+=========
+Hong Kong
+=========
+
+Configuration
+=============
+
+:ref:`Install <general/install>` the :guilabel:`Hong Kong - Accounting` module to
+get the features of the HK localization.
+
+.. image:: hong_kong/hk_01.png
+   :align: center
+
+Add FPS QR Codes to invoices
+============================
+
+Activate QR Code feature
+------------------------
+
+Go to :menuselection:`Accounting --> Configuration --> Settings`
+and activate the **QR Code** feature.
+
+.. image:: hong_kong/hk_02.png
+   :align: center
+
+Configure your FPS bank account
+-------------------------------
+
+Go to :menuselection:`Contacts --> Configuration --> Bank Accounts`,
+select a Account Holder with Hong Kong as country, open your FPS bank
```

Comment:
```suggestion
select an account holder with Hong Kong as country, open your FPS bank
```
I no longer see where to "open your FPS bank account and fill in the "**FPS Type** and **FPS ID/Phone Number/Email Address** fields. Is there any configuration needed to be able to see it? It worked on the runbot you sent but not when I try to reproduce it in another runbot.

Reply:
The feature is not yet deployed to master therefore is not existed yet on other Runbot.